Blog post: call for volunteers to join LPGCs (#1571)
carpentries/carpentries.org: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md

---
layout: page
authors: ["Toby Hodges"]
teaser: "Guide the development of Software Carpentry, Library Carpentry, or Data Carpentry"
title: "Calling for Volunteers to join a Lesson Program Governance Committee"
date: 2023-01-19
time: "09:00:00"
tags: ["Governance", "Community", "Data Carpentry", "Library Carpentry", "Software Carpentry"]
---

The Carpentries has three lesson programs, [Software Carpentry][swc], [Library Carpentry][lc], and [Data Carpentry][dc].

> In November, I published [a post describing the governance structure of each lesson program, and outlining a plan to 

Reviewer: I would link the word 'post' not the entire sentence.

introduce more community governance in the form of _Lesson Program Governance Committees (LPGCs)_][lpgc-planning-post]. 

> **Today, we open the call for volunteers to form these new committees.**

Reviewer: ```suggestion
**Today, 19 January we open the call for volunteers to form these new committees.**
```

program at conferences and other relevant events. They will also report and make recommendations to [the Executive 
Council][exco], coordinate with the [Curriculum Advisory Committee(s) (CAC(s))][cac] for their lesson program, and 
communicate with the wider lesson program community.

Especially as these committees are being established for the first time, they will be given a large degree of 
self-determination: in accordance with [the Lesson Program Governance Policy][lp-policy], the Governance Committees will 
have the power to decide on their own terms, e.g. the length of member term limits, the method by which new members will be 
selected (e.g. elected or approved), how they overlap/interact with Curriculum Advisors, request additional powers from the 
Executive Council, etc.

As with other committees in The Carpentries, each LPGC will include a Chair and Secretary. These officers will hold 
additional responsibilities alongside those of a regular LPGC member, and will receive additional onboarding to ensure they 
are well-prepared to take on the roles. The responsibilities associated with these officer roles will be almost identical to 

> [those of the Chair and Secretary of Curriculum Advisory Committees][cac-officers].

Reviewer: ```suggestion
[the Chair and Secretary of Curriculum Advisory Committees][cac-officers].
```

security and privacy policies) e.g. data to inform committee discussions concerning the health of the lesson program.
- Comprehensive documentation as a handbook for Lesson Program Governors.

## Who Should Apply?
You should!
We would like committees to be broadly representative of their lesson program community, which means including perspectives 
from every community role. 
**Maintainers** have a vital role to play, bringing their experience of handling contributions to lessons and interacting 
with the community on GitHub. 
The voice of **Instructors** will be essential, to inform discussions based on experience gained from teaching workshops and 
interacting with Learners. 
**Workshop Hosts** can speak to the value of the lesson program to their communities, and how Carpentries workshops fit into 
the bigger picture of training and professional development in their context. 
**Curriculum Advisors** could provide a vital bridge between the governance of curriculum and the lesson program as a whole.
If you are reading this, would like to apply, but do not fit any of the roles mentioned above, you should still do so! If 

> you have any questions or concerns, please [reach out to me](mailto:tobyhodges@carpentries.org).

Reviewer: ```suggestion
you have any questions or concerns, please [contact me](mailto:tobyhodges@carpentries.org).
```

## What is the Time Requirement?
As this is the first time that LPGCs are being established, we have no information about the time required for this role. 
However, committees are expected to meet quarterly, with a minimum of 1 hour of preparation/follow-up work (probably 2 for 
the committee Chair).
Taking into account other time that may be taken up by communication and other tasks, you can expect LPGC membership to 
require a minimum of 3 hours every three months.

## How to Apply

> You can [volunteer to join a Lesson Program Governance Committee by filling out this short form][volunteer-application].

Reviewer: ```suggestion
You can volunteer to join a Lesson Program Governance Committee by filling out [this short form][volunteer-application].
```
